ci-rustc.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash -ex
+
+repo_root=$(readlink -f $(dirname "${BASH_SOURCE[0]}"))
```

Comment:
Why was this file added? It looks like it's perhaps something added by mistake?

Reply:
Yes that was a mistake. Removed it